changelog.d/19716_doc_vrl_get_timezone_name.enhancement.md

```diff
@@ -0,0 +1,3 @@
+New VRL function `get_timezone_name` to return the configured/resolved IANA timezone name.
```

Comment:
Could we drop this file @klondikedragon ? It'll actually pull in the VRL changelog entry when we generate the Vector changelog so we don't need this one too.